### ISSUE TITLE
Unescaped left brace in regex is deprecated, passed through in regex

### ISSUE THREAD

**Issue by fl0rianB** [ftilmann/latexdiff#97]
I work on an up to date Debian Testing and still get the the deprecation warnings. Is their a problem with my configuration ?
`latexdiff --version` produces these errors: 

```
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\includeonly{ <-- HERE (.*?)}/ at /usr/bin/latexdiff line 1572.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/(^(?:[^%\n]|\\%)*)\\bibliography{ <-- HERE (.*?)}/ at /usr/bin/latexdiff line 1607.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/(^(?:[^%\n]|\\%)*)\\subfile{ <-- HERE (.*?)}/ at /usr/bin/latexdiff line 1622.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/(?<!\\)\\{ <-- HERE / at /usr/bin/latexdiff line 2351.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{ <-- HERE MATHMODE}((?:(.(?!(?<!\\)\&|\\\\))*)?)\\end{MATHMODE}/ at /usr/bin/latexdiff line 2676.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{MATHMODE}((?:(.(?!(?<!\\)\&|\\\\))*)?)\\end{ <-- HERE MATHMODE}/ at /usr/bin/latexdiff line 2676.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{ <-- HERE MATHMODE}(.*?)\\end{MATHMODE}/ at /usr/bin/latexdiff line 2678.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{MATHMODE}(.*?)\\end{ <-- HERE MATHMODE}/ at /usr/bin/latexdiff line 2678.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\end{ <-- HERE SQUAREBRACKET}/ at /usr/bin/latexdiff line 2759.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\begin{ <-- HERE SQUAREBRACKET}/ at /usr/bin/latexdiff line 2760.
This is LATEXDIFF 1.1.1  (Algorithm::Diff 1.15 fast, Perl v5.24.1)
  (c) 2004-2015 F J Tilmann
```

**Comment by ftilmann** [comment]
Most of these have been fixed in release 1.2.0 (with a few stray ones only fixed on the unreleased github version) - these are just warnings, though and you can ignore them. 
I don't know why Debian testing is still on 1.1.1 - you would have to take this up with their latexdiff package maintainer.

**Comment by kentfredric** [comment]
> these are just warnings, though and you can ignore them

Not on Perl 5.26 you can't. They're fatal there.
